proposals/lip-replace_static_fee_system_by_dynamic_fee_system.md

```diff
@@ -0,0 +1,358 @@
+
+### Details of current proposal
+
+Each time a transaction is broadcast into the network and eventually included into a block, resources are consumed. In the case of the Lisk network, there are two types of resources being used depending on the type of transaction:
+
+1. Bandwidth in the network, dynamic memory and disk storage in the nodes : **all transaction types**.
+
+2. Namespace: **delegate registration and dapp registration transactions**.
+
+The new fee system takes into account the usage of these resources to give a minimum required fee for a certain transaction to be valid. We define this minimum fee per transaction as:
+
+`trs.minFee = trs.nameFee + minFeePerByte * sizeof(trs)`
+
+`trs.minFee` is given in LSK, `minFeePerByte` stands for the minimum fee per byte, given in LSK/byte, `sizeof(trs)` is the size of the transaction object in bytes and `trs.nameFee` is a parameter, given in LSK, that takes different values depending on the transaction type:
+
+1. For transaction types 0, 1, 3 and 4 (balance transfer, second passphrase registration, voting and multisignature registration transactions), `trs.nameFee` = 0.
+
+2. For transaction types 2 and 5 (delegate registration and dapp registration transactions), `trs.nameFee` > 0.
+
+#### Minimum fee per byte: `minFeePerByte`
+
+The right choice of the minimum fee per byte, `minFeePerByte`, will have a great impact in the network. If the minimum required fee were too high, users would be discouraged from using the network. If the value were too low, attackers would be able to broadcast many valid transactions at a very low cost and congest the network. Regarding the last issue, in this proposal, it is assumed that adequate measures are in place to prevent this from occurring. These measures are mostly implemented in the peer-to-peer (P2P) layer and in the transaction pool. Later in this section, the impact on these modules will be commented more in detail.  
+
+With the assumption of having these “anti-spam” measures in the previously mentioned modules, it would be interesting to consider how these resources mentioned above are used by all the transaction types and the cost implied. Unfortunately, the calculation of the exact cost of these resources is very complex and depends on several unrelated points as, hardware specifications, connection bandwidth, software platform, cost of electricity, Lisk token volatility, etc. An accurate research about this would be extremely complex and it is out of the scope of this proposal. However, it is still important to study the price of the different transactions and the price of broadcasting several of them depending on the value of `minFeePerByte`:  
+
+- [Appendix A](#Appendix-A:-Minimum-fee-implications) has an overview of minimum transaction fees depending on different values of `minFeePerByte` and the estimated direct implications for the network.
```

Comment:
The link seems to be incorrect. Also for the other occurrences of appendix links.

Reply:
corrected all the appendix links